src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -155,10 +156,19 @@ public async override Task<P.CreateInstanceResponse> StartInstance(P.CreateInsta
                         InstanceId = instanceId,
                     };
                 }
-                catch (InvalidOperationException)
+                catch (OrchestrationAlreadyExistsException)
                 {
                     throw new RpcException(new Status(StatusCode.AlreadyExists, $"An Orchestration instance with the ID {request.InstanceId} already exists."));
                 }
+                catch (Exception ex)
```

Comment:
I feel like the following would be ideal
```suggestion
                catch (OrchestrationAlreadyExistsException)
                {
                    throw new RpcException(new Status(StatusCode.AlreadyExists, $"An Orchestration instance with the ID {request.InstanceId} already exists."));
                }
                ```suggestion
                catch (InvalidOperationException ex) when (ex.Message.EndsWith("already exists.")) // for older versions of DTF.AS and DTFx.Netherite
                {
                    
                    throw new RpcException(new Status(StatusCode.AlreadyExists, $"An Orchestration instance with the ID {request.InstanceId} already exists."));
                }
                catch (Exception ex)
```

Reply:
Maybe, though it won't ever be used for DT.AzureStorage if we release these changes together. It would really only be applicable for old versions of Netherite.

Reply:
Oh, right - we control the DF.AS version. That mitgates this somewhat.

Reply:
@cgillum: do you have any strong reason _not_ to incorporate this? Even if it just affects Netherite, that seems sufficient reason to me.

Reply:
@cgillum: In that case I added it :-). Thanks!